CMSgov/mint-app
# [task: EASI-3923] Toggle conditions on pdf export

# EASI-3923


## Changes and Description

- Added a context wrapper on readonly views to manage state when pdf export is enabled
- Updated `CollapsableLink` to ingest state of context and toggle context when pdf export is enabled
  - Added additional prop to disable context override if doesn't need to export on export
  - Default to false
- Updated `ShareExport` to trigger context when beginning pdf export and after pdf is exported



## How to test this change

Verify that the conditional questions are always expanded when exporting PDF
Verify that they return to the toggle state they were in before print

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
